app/Console/Kernel.php

```diff
@@ -72,6 +74,11 @@ protected function schedule(Schedule $schedule)
             ->hourly()
             ->onOneServer()
             ->withoutOverlapping();
+
+        $schedule->command(DeleteTemporaryZipFiles::class)
+            ->hourly()
+            ->onOneServer()
```

Comment:
@mgonzalezs this needs to run on each server, so i believe it is incorrect to use onOneServer?

Reply:
You are right. Fixed